setup.cfg

```
@@ -1,4 +1,62 @@
-# This file should be used as the pep8/pycodestyle and autopep8 config
+###############################################################################
+# Setuptools
+###############################################################################
+
+[metadata]
+name = WebAppDIRAC
+description = WebAppDIRAC is a prortal for the DIRAC software.
+long_description = file: README.rst
+url = https://github.com/DIRACGrid/WebAppDIRAC/
+license = GPL-3.0-only
+license_files = LICENSE
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: GNU General Public License v3 (GPLv3)
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Topic :: Scientific/Engineering
+    Topic :: System :: Distributed Computing
+
+[options]
+python_requires = >=3.8
+package_dir=
+    =src
+packages = find:
+
+install_requires =
+    dirac
+    diraccfg
+    #autopep8 ==1.3.3
+    coverage
+    requests >=2.9.1
+    six
+    #mock >=1.0.1
+    #pylint >=1.6.5
+    #pytest >=3.6
+    #pytest-cov >=2.2.0
+    #pytest-mock
+    #tornado >=5.0.0
```

I'm not sure if this is correct but you'll need to include the package data:
```suggestion

zip_safe = False
include_package_data = True

[options.package_data]
* =  WebApp/static
```

You can check by running `python setup.py sdist bdist_wheel` and unzipping the files which are generated to make sure everything you expect is included.

---

protested with `include_package_data`, with `options.package_data ` and without, the result is no different. in all cases js, tpl and others are available in the archive